src/components/MessagesCard.vue

```diff
@@ -1,12 +1,27 @@
 <template>
-    <div
-      class="card bg-white shadow-md border border-gray-200 rounded-lg w-3/4 p-4 sm:p-6 lg:p-8 dark:bg-gray-800 dark:border-gray-700"
-    >
-    <img class="inline object-cover w-16 h-16 mr-2 rounded-full" src="https://images.pexels.com/photos/2589653/pexels-photo-2589653.jpeg?auto=compress&cs=tinysrgb&h=650&w=940" alt="Profile image"/>
-     <p class="mt-5">
-       </p>
-      <slot></slot>
-    </div>
+  <div
+    class="card bg-white shadow-md border border-gray-200 rounded-lg w-3/4 p-4 sm:p-6 lg:p-8 dark:bg-gray-800 dark:border-gray-700"
+  >
+ <nav class="flex items-center justify-between h-full p-3 m-auto">
+  <span>My Name</span>
+  <div class="relative">
+    <button id="user-menu" aria-label="User menu" aria-haspopup="true">
+        <img class="w-14 h-14 rounded-full" src="https://t3.ftcdn.net/jpg/03/46/83/96/360_F_346839683_6nAPzbhpSkIpb8pmAwufkC7c5eD7wYws.jpg" />
+    </button>
+    <div id="user-menu-dropdown" class="absolute right-0 w-48 mt-2 origin-top-right rounded-lg shadow-lg top-10 menu-hidden">
+      <div class="p-4 bg-white rounded-md shadow-xs" role="menu" aria-orientation="vertical" aria-labelledby="user-menu">
+        <a href="#" class="block px-6 py-2 mb-2 font-bold rounded fas fa-user" role="menuitem">My profile</a>
+        <a href="#" class="block px-6 py-2 mb-2 font-bold rounded fas fa-bell" role="menuitem">Notifications</a>
+        <a href="#" class="block px-6 py-2 mb-2 font-bold rounded fas fa-envelope" role="menuitem">Inbox</a>
+        <a href="#" class="block px-6 py-2 font-bold rounded fas fa-sign-out-alt" role="menuitem">Logout</a>
+      </div>
+    </div> 
+  </div> 
+</nav> 
```

Comment:
@Queen-of-Diamonds looks like the formatting's gone a bit off.  Try `shift+alt+F` - see if that autofixes it, then commit as `fix(formatting): MessagesCard` and push to here.